Tests/LogicTests/SNTFileInfoTest.m

```diff
@@ -132,6 +132,65 @@ - (void)testBundle {
   XCTAssertEqualObjects([sut bundlePath], @"/Applications/Safari.app");
 }
 
+- (void)testAncestorBundle {
+  NSString *nestedBundleBinary =
+      @"/Applications/Safari.app/Contents/XPCServices/"
+      @"com.apple.Safari.BrowserDataImportingService.xpc/"
+      @"Contents/MacOS/com.apple.Safari.BrowserDataImportingService";
```

Comment:
Could do, though testing the Ancestor Bundle idea wouldn't work because LogicTests.xctest the highest bundle is the path :/